samples/text_panes/components/prism.js

```diff
@@ -0,0 +1,33 @@
+AFRAME.registerComponent("prism", {
```

Comment:
Not entirely sure where this is being use, can we use a more descriptive name?

Reply:
My bad, the naming was ambiguous, changed it to text-box